Images/map/in-person.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can you please rebase this feature branch on the current dev branch?  Lemme know if you have issues with conflicts and I'll try to assist.